wgsl/index.bs

```diff
@@ -1527,6 +1527,8 @@ These types cannot be spelled in WGSL source. They are only used by [=type check
 
 A type that is not an abstract numeric type nor contains an abstract numeric
 type is called <dfn noexport>concrete</dfn>.
+Similarly, a type that is an abstract numeric type or contains an abstract
+numeric type is called an <dfn noexport>abstract type</dfn>.
```

Comment:
`<dfn for=type noexport>abstract</dfn> type` works, then use `[=type/abstract|abstract type=]`

Reply:
I'd flip this around, because the 'concrete' definition is in the negative.

> A type is `<dfn for=type noexport>abstract</dfn>` if it is an abstract numeric type or contains an abstract numeric type.  A type is `concrete` if it is not `abstract`.